yt/visualization/fixed_resolution.py

```diff
@@ -561,6 +563,7 @@ def __init__(self, data_source, bounds, buff_size, antialias = True,
         FixedResolutionBuffer.__init__(self, data_source, bounds, buff_size, antialias, periodic)
 
     def __getitem__(self, item):
+        item = self.data_source._determine_fields(item)[0]
```

Comment:
As a quick note, I spent a few minutes thinking about what happens if this results in a length > 0 array, but (correct me if I'm wrong) this should never happen, and this is perfectly cromulant code.

Would it be worthwhile to semi-enforce this by doing `item, = ...` ?  (Probably not?)